ethereum_bridge/src/protocol/transactions/votes/update.rs

```diff
@@ -376,6 +377,7 @@ mod tests {
     #[test]
     fn test_apply_duplicate_votes() -> Result<()> {
         let mut wl_storage = TestWlStorage::default();
+        test_utils::init_default_storage(&mut wl_storage);
```

Comment:
I think we never query voting powers from storage in `apply`, so this shouldn't be necessary, unless I'm missing another reason why you've added this here? well, either way, it doesn't hurt to be safe by preemptively initializing storage with  validator data

Reply:
I think this is needed for the call to `total_consensus_stake_key_handle().set(...` in `TallyParams::setup` that read PoS params